frontend/apps/reader/readerui.lua

```diff
@@ -324,6 +322,11 @@ function ReaderUI:init()
     for _,v in ipairs(self.postInitCallback) do
         v()
     end
+
+    -- After initialisation notify that document is loaded
+    -- CREngine only reports correct page count after rendering is done
+    -- Need the same event for PDF document
+    self:handleEvent(Event:new("PostRenderDocument", self.doc_settings))
```

Comment:
I suggest we keep or remove `PostRenderDocument` event and add a `ReaderReady` event at this line instead :)